<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiltersDialog</class>
 <widget class="QDialog" name="FiltersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableView" name="filterTableView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="buttonsLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QToolButton" name="addFilterButton">
             <property name="toolTip">
              <string>New Filter</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../glogg.qrc">
               <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="removeFilterButton">
             <property name="toolTip">
              <string>Delete Filter</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../glogg.qrc">
               <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="upFilterButton">
             <property name="toolTip">
              <string>Move Filter Up</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../glogg.qrc">
               <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="downFilterButton">
             <property name="toolTip">
              <string>Move Filter Down</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../glogg.qrc">
               <normaloff>:/images/down.png</normaloff>:/images/down.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../glogg.qrc"/>
 </resources>
 <connections/>
</ui>
